submissions/JuliaNovak/TJSW-pre-oop/index.js

```diff
@@ -0,0 +1,50 @@
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: "dog",
+   name: "Toby",
+   gender: "male",
+   legs: 4,
+   hands: 0,
+   saying: "woof-woof!",
+};
+
+const cat = {
+   species: "cat",
+   name: "Lily",
+   gender: "female",
+   legs: 4,
+   hands: 0,
+   saying: "meow!",
+};
+
+const woman = {
+   species: "human",
+   name: "Linda",
+   gender: "female",
+   legs: 2,
+   hands: 2,
+   saying: "What a pretty kitten!",
+};
+
+const man = {
+   species: "human",
+   name: "Jim",
+   gender: "male",
+   legs: 2,
+   hands: 2,
+   saying: "Heyyo!",
+};
+
+const catWoman = Object.create(cat);
+catWoman.species = "catWoman";
+(catWoman.name = "Ro"), (catWoman.legs = 2), (catWoman.hands = 2);
```

Comment:
Interesting approach, but it is fine to write this in 3 lines. It's not like it's some code that could be "skipped", it's an important value assigning that describes one of the objects.

Reply:
is there another way to do it?

Reply:
Just write 3 separate lines)

Reply:
I mean, is there a way to do it without using Object.create?:)